app/pages/neworg/findorg/template.njk

```diff
@@ -28,7 +28,7 @@
         <form method="post">
           <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
           <div class="nhsuk-grid-row">
-            <div class="nhsuk-grid-column-one-third">
+            <div class="nhsuk-grid-column-one-half">
               {{ textField({
                 question: questions[0]
```

Comment:
should the div always be 2 thirds and some prop we can pass into the text field to determine how large this should be?

Reply:
this is the only thing in this div - it is going from 1/3 to 1/2 so the validation errors don't look stupid

Reply:
it is 1/2 of the 2/3 width

Reply:
I think the div should always be 2/3 and not use that to determine the size of the textField. So the validation error doesn't look stupid. Can you provide a class to textField like
`"classes": "nhsuk-input--width-10"` to keep it small?